common/views/components/RelevanceRater/RelevanceRater.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import styled from 'styled-components';
+import { classNames } from '../../../utils/classnames';
+import {
+  track,
+  type RelevanceRating as RelevanceRatingType,
+} from '../SearchLogger/SearchLogger';
+
+const RelevanceRaterStyle = styled.div.attrs(props => ({
+  className: classNames({
+    flex: true,
+  }),
+}))``;
+
+const RelevanceRating = styled.button.attrs(props => {})``;
+
+type Props = {| position: number, id: string |};
+
+function createEvent(rating: RelevanceRatingType) {
+  return {
+    service: 'relevance_rating',
+    event: 'Rate Result Relevance',
+    rating: rating,
+  };
+}
+
+const RelevanceRater = ({ id, position }: Props) => {
+  return (
+    <RelevanceRaterStyle>
+      <RelevanceRating
```

Comment:
Could we find a simple way to indicate whether `1` or `4` is most relevant?

Reply:
The interface / wording etc really needs work. @hthair  does have a key with with words that make more sense.

This is just to see if it works.

FYI 1 is irrelevant / 4 relevant.

Reply:
We'll be making a ticket for this and we can capture all the piece that need doing.